packages/app-check/android/src/main/java/io/capawesome/capacitorjs/plugins/firebase/appcheck/FirebaseAppCheckPlugin.java

```diff
@@ -45,9 +45,18 @@ public void error(String message) {
     @PluginMethod
     public void initialize(PluginCall call) {
         try {
-            boolean debug = call.getBoolean("debug", false);
+            Boolean debugTokenBoolean = call.getBoolean("debugToken", false);
```

Comment:
You should still check the `debug` option. Otherwise this would be a breaking change. The property is just deprecated.

Reply:
Yes, we can't publish a breaking change right now because we have to wait for the next Capacitor major release to keep the versions in sync. That's why we should just deprecate the `debug` option and still use it as fallback if `debugToken` is not set.